XenModel/Actions/ZipStatusReportAction.cs

```diff
@@ -62,13 +62,29 @@ public class ZipStatusReportAction : AsyncAction
 
         private long bytesToCompress = 1;
 
-        public ZipStatusReportAction(string tempFolder, string destFile)
-            : base(null, Messages.BUGTOOL_SAVING, Messages.BUGTOOL_SAVING, true)
+        /// <summary>
+        ///  Constructor to zip downloaded file action
+        /// </summary>
+        /// <param name="tempFolder">empFolder to store the downloaded logs</param>
+        /// <param name="destFile">The target file to store the compress result</param>
+        /// <param name="suppressHistory">Whether suppress history in the event notification</param>
+        public ZipStatusReportAction(string tempFolder, string destFile, bool suppressHistory)
+            : base(null, Messages.BUGTOOL_SAVING, Messages.BUGTOOL_SAVING, suppressHistory)
         {
             _inputTempFolder = tempFolder;
             _destFile = destFile;
         }
 
+        /// <summary>
+        /// Construcor, set suppressHistory to ture
+        /// </summary>
+        /// <param name="tempFolder">tempFolder to store the downloaded logs</param>
+        /// <param name="destFile">The target file to store the compress result</param>
+        public ZipStatusReportAction(string tempFolder, string destFile):this(tempFolder,destFile,true)

```

Comment:
There is no need for a separate constructor, we can set the default vlaue of the suppressHistory parameter to true in the previous constructor: 
```public ZipStatusReportAction(string tempFolder, string destFile, bool suppressHistory = true)```